cluster-applications/gp-cluster-applications/values/values-play.yaml

```diff
@@ -210,3 +210,30 @@ applications:
       automated:
         prune: true
         selfHeal: true
+
+  ####################### Grafana Instance ######################
+  grafana-instance:
+    name: grafana-instance
+    enabled: true 
+    argoProject: gepaplexx
+    destination:
+      namespace: grafana-operator-system
+      create: true
+    source:
+      repoURL: "https://gepaplexx.github.io/gp-helm-charts/"
+      targetRevision: "*"
+      chart: "gp-grafana-instance"
+      helm:
+        parameters:
+          - name: "grafana.auth.admin.user"
+            value: "admin"
+          - name: "grafana.auth.admin.password"
+            value: "TODO-password"
```

Comment:
passwort und token sollten nicht cleartext in einem public Repo stehen. 
Hier sollten wir entweder über SealedSecrets arbeiten (beim Passwort wird das vermutlich so bleiben). bzw. beim Token, nachdem dieser Abhängig von der Installation ist, könnte man wie bei der K10-Instanz sagen, dass dies ein manueller Schritt beim Cluster-Setup ist und in der ArgoCD Application einen "ignoreDifferences" Block hinzufügen, damit Argo die Änderung nicht Rückgängig macht.

Reply:
passwort wird jetzt generiert